docs/azure_arc_jumpstart/azure_arc_sqlsrv/levelup/_index.md

```diff
@@ -0,0 +1,327 @@
+
+2. Fill in the parameter values as shown below and leave other values default, click on Review + create to continue deployment.
+
+  ![Fill in parameters](azure-portal-deployment-params.png)
+
+3. Review all the values and click on Create to deploy level up lab environment.
+
+  ![Create deployment](azure-portal-deployment-create.png)
+
+4. This deployment takes around 15 minutes. Monitor deployment progress and make sure deployment is successful. 
+
+  ![Review deployment progress](azure-portal-deployment-progress.png)
+
+5. When deployment is successful click on Go to resource group to add NSG rules to allow RDP access.
+
+  ![Deployment complete](azure-portal-deployment-complete.png)
+
+6. This deployment contains 9 resources including ArcBox-Client VM that will be used as part of the lab modules.
+
+  ![Deployed resources](azure-portal-deployment-resources.png)
+
+### (Option 2): Deploy using Azure CLI
+
+1. Open PowerShell window and change directory to **C:\\** and create **ArcSqlLevelup** sub folder and change directory to **C:\ArcSqlLevelup** folder.
+
+2. Clone the Azure Arc Jumpstart repository using command below using PowerShell window on your computer or Azure Cloud Shell.
+
+  ```shell
+  git clone -b lu_arc_sql https://github.com/microsoft/azure_arc.git
+  ```
+
+  ![Git clone Azure Arc repository](git-clone-lu-arc-sql.png)
+
+3. Open **azuredeploy.parameters.json** file located under **C:\ArcSqlLevelup\azure_arc\azure_jumpstart_arcbox_levelup\ARM** folder in a notepad or choice of your editor.
+
+  ![Parameters file location](azure-arm-parameters.png)
+
+4. Update **azuredeploy.parameters.json** parameter file by providing values for the highlighted parameters below.
+
+  ```shell
+  {
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "spnClientId": {
+        "value": "<your service principal client id>"
+      },
+      "spnClientSecret": {
+        "value": "<your service principal secret>"
+      },
+      "spnTenantId": {
+        "value": "<your spn tenant id>"
+      },
+      "windowsAdminUsername": {
+        "value": "arcdemo"
+      },
+      "windowsAdminPassword": {
+        "value": "ArcPassword123!!"
+      },
+      "logAnalyticsWorkspaceName": {
+        "value": "<your unique Log Analytics workspace name>"
+      },
+      "deployBastion": {
+        "value": false
+      }
+    }
+  }
+  ```
+
+5. Create resource group using command below.
+
+  ```shell
+  az group create -n ArcSql-Levelup -l eastus
+  ```
+
+  ![Create resource group](azure-arm-group-create.png)
+
+6. Deploy ARM template using command below. Please make sure you change the file path in case if you chose different folder for git clone.
```

Comment:
```suggestion
6. Deploy the ARM template using the command below. Please make sure you change the file path in case you chose a different folder when cloning the repository.
```